jmx-scraper/src/main/java/io/opentelemetry/contrib/jmxscraper/config/JmxScraperConfig.java

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jmxscraper.config;
+
+import static io.opentelemetry.contrib.jmxscraper.util.StringUtils.isBlank;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/** This class keeps application settings */
+public class JmxScraperConfig {
+
+  static final String SERVICE_URL = "otel.jmx.service.url";
+  static final String CUSTOM_JMX_SCRAPING_CONFIG = "otel.jmx.custom.scraping.config";
+  static final String TARGET_SYSTEM = "otel.jmx.target.system";
+  static final String INTERVAL_MILLISECONDS = "otel.jmx.interval.milliseconds";
+  static final String METRICS_EXPORTER_TYPE = "otel.metrics.exporter";
+  static final String EXPORTER_INTERVAL = "otel.metric.export.interval";
+  static final String REGISTRY_SSL = "otel.jmx.remote.registry.ssl";
+
+  static final String OTLP_ENDPOINT = "otel.exporter.otlp.endpoint";
+
+  static final String JMX_USERNAME = "otel.jmx.username";
+  static final String JMX_PASSWORD = "otel.jmx.password";
+  static final String JMX_REMOTE_PROFILE = "otel.jmx.remote.profile";
+  static final String JMX_REALM = "otel.jmx.realm";
+
+  static final List<String> AVAILABLE_TARGET_SYSTEMS =
+      Arrays.asList(
```

Comment:
Should wrap this with `Collections.unmodifiableList()` so that the list cannot accidentally be modified.